frontend/src/components/Card/Card.module.scss

```diff
@@ -7,13 +7,13 @@
   display: flex;
   margin: 0 5px;
   flex-direction: column;
-  cursor: pointer;
   transition: transform 0.3s;
   border-radius: 5px;
-}
-
-.card:hover {
-  transform: scale(1.04);
+  
+  &:hover {
+    cursor: pointer;
```

Comment:
idk what the difference between these is but cool

Reply:
just sass things